src/sentry/relay/config/__init__.py

```diff
@@ -250,7 +250,7 @@ def _get_project_config(
 
     if features.has("organizations:metrics-extraction", project.organization):
         config["sessionMetrics"] = {
-            "version": 1,
+            "version": 2 if features.has("projects:release-health-anr-extraction", project) else 1,
```

Comment:
The consequence of this version bump is that external relays will stop extracting session metrics until they are upgraded, and will forward session payloads to Sentry's own relays instead (thereby increasing the load on our system). Is this the behavior that we want? Or could we live with external relays not extracting ANR metrics until they update?